environment/OMAS_process.m

```diff
@@ -780,32 +792,26 @@
     [graph, id_to_index] = create_graph(agents);
     size_comp = networkComponents(graph.p);
     
-    concensus_data = [];
-    % Remaining questions
-        % The graph is not directed, correct?
-        % Do we update the agents' values after each consus step?
-        % What is the union/groupset if statement for?
-        % Errors with calc_ci_weights_ver3
-        % It is easy to implement a seperate observation and communication connection check
-        
-    
     %% Compute first values
     % Y_k+1 = Y + I;
     % y_k+1 = y + i;
     % det ratio?
```

Comment:
One question - Why is the first step different, and is this supposed to be Y = Y + sum(I for each neighbor)?